### ISSUE TITLE
State variable (uint only?) initialisation includes dead code

### ISSUE THREAD

**Issue by axic** [ethereum/solidity#1268]
```
contract A {
    uint a = 1;
}
```

always seem to create a NOP (in the form of PUSH+POP):

```
  PUSH 1            1
  PUSH 0            uint a = 1
  PUSH 0            uint a = 1
  POP           uint a = 1
  SSTORE            uint a = 1
```

**Comment by chriseth** [comment]
Fixed by https://github.com/ethereum/solidity/issues/887

**Comment by axic** [comment]
Correction: will be fixed when #887 is fixed :smiley:
